inlong-audit/audit-proxy/src/main/java/org/apache/inlong/audit/source/ServerMessageHandler.java

```diff
@@ -147,6 +151,12 @@ private AuditReply handleRequest(AuditRequest auditRequest) throws Exception {
         List<AuditMessageBody> bodyList = auditRequest.getMsgBodyList();
         int errorMsgBody = 0;
         for (AuditMessageBody auditMessageBody : bodyList) {
+            long msgDays = messageDays(auditMessageBody.getLogTs());
+            if (msgDays >= this.msgValidThreshold) {
+                LOGGER.info("Discard the data as it is from" + msgDays +
```

Comment:
It is recommended to adjust the log of discarded data to warn level

Reply:
Modified.